src/main/java/com/desiato/puresynth/mappers/DTOMapper.java

```diff
@@ -0,0 +1,20 @@
+package com.desiato.puresynth.mappers;
+
+import com.desiato.puresynth.dtos.UserResponseDTO;
+import com.desiato.puresynth.models.User;
+import org.springframework.stereotype.Component;
+
+
+@Component
+public class DTOMapper {
+
+    public UserResponseDTO toUserResponseDTO(User user) {
```

Comment:
A good simple name could be just `map` or `toDto`. Then in the controller you have `userDtoMapper.map(...)`